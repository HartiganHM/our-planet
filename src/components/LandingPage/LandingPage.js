import React from 'react';
import './LandingPage.css';

const LandingPage = () => {
  return <div className="LandingPage">I am a Landing Page!</div>;
};

export default LandingPage;
