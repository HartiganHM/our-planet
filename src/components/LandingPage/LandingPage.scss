@import '../../styles/variables.scss';
@import '../../styles/mixins.scss';

.LandingPage {
  height: 100vh;
  margin: auto;
  display: table;
  top: 0;
  background-size: cover;
  color: $accent-color-dark;
  .inner {
    display: table-cell;
    vertical-align: middle;
    width: 100%;
    max-width: none;
    .content {
      @include maxWidth(50);
      margin: 0 auto;
      text-align: center;
    }
  }
}