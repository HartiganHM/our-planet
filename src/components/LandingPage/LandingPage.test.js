import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from './LandingPage';

describe('LandingPage tests', () => {
  it('Should match the snapshot', () => {
    const renderedLandingPage = shallow(<LandingPage />);

    expect(renderedLandingPage).toMatchSnapshot();
  });
});
