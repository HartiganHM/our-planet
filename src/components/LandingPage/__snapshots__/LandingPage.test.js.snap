// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage tests Should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LandingPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="inner"
>
        <div
                className="content"
        />
</div>,
      "className": "LandingPage",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="content"
/>,
        "className": "inner",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "content",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="inner"
>
          <div
                    className="content"
          />
</div>,
        "className": "LandingPage",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="content"
/>,
          "className": "inner",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "content",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
